pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -816,7 +819,7 @@ self: super: {
   # See https://github.com/haskell-hvr/cryptohash-sha512/pull/3
   # , https://github.com/haskell-hvr/cryptohash-sha512/issues/4
   # and https://github.com/haskell-hvr/cryptohash-sha512/pull/5
-  cryptohash-sha512 = doJailbreak super.cryptohash-sha512;
+  cryptohash-sha512 = dontCheck (doJailbreak super.cryptohash-sha512);
```

Comment:
Why did you add `dontCheck` here?  `cryptohash-sha512` appears to currently be building even with the tests enabled.

Reply:
Oh, I see the explanation is in the commit.

Could you add this explanation in a comment above this line?  Also, ideally link to an issue upstream where the nix maintainers can track the status of this.  (Although it seems like cryptohash-sha512 has been somewhat abandoned?)

> base16-bytestring had new epoch (1.0) 3 month ago which fixed the old return type to normal `Either String ByteString`. `cryptohash-sha512` is stale for ~10-11 months and testsuite does not support the new `base16-bytesting` releases.

Ah, I guess this new version of base16-bytestring hasn't made its way to the Stackage nightlies yet, which is why cryptohash-sha512 doesn't yet need `dontCheck`: https://github.com/commercialhaskell/stackage/issues/5649

Reply:
Yes. It is a still working remnant of an old story.

Since it is a well-known public story I would refresh: 
When `vincenthz` was preamptively deprecating his old working `cryptohash` in favor of the new not fully formed `cryptonite`, somewhere in that time they _talked about something_. `hvr` forked the `cryptohash` and split the project into `cryptohash-*` family of more lightweight packages. In parallel, there was a debacle with the `snoyman`. And with everything, all those went to public boards and records.

No reason to say with the number of things `hvr` has on the plate. The whole `cryptohash-*` family pretty much got abandoned right after and these years rolled on merging of PRs from `phadej` & drive-by people. But it seems `hvr` stopped appearing in the https://github.com/haskell-hvr ~9-10 months ago, IDK his patterns, maybe he goes in the once a year. But probably his participation understandably dropped.

 It seems to many of those projects only he has merge rights, `cryptohash-*` family is most probably RIP as a bunch of other projects there, so you probably can await others from that group.

Reply:
Ah, thanks for the explanation!

Reply:
That is probably too opinionated and too in depth comment about a very sensitive issue from me, who was not participating in all that. Just wanted to refresh your memory on that `cryptohash-*` family and why there is a `hs-cryptohash` and `cryptohash-*` and `cryptonite` and why they use the same hierarchical module names. And that it is probable that you receive some set of PRs for projects that are variations on the theme.